app/services/families/invite.rb

```
@@ -0,0 +1,125 @@
+
+      true
+    rescue ActiveRecord::RecordInvalid => e
+      handle_record_invalid_error(e)
+      false
+    rescue Net::SMTPError => e
+      handle_email_error(e)
+      false
+    rescue StandardError => e
+      handle_generic_error(e)
+      false
+    end
```

RuboCop (1.81.1)

[convention] Method has too many lines. [17/10]

(Metrics/MethodLength)